backend/ethereum/client/init_test.go

```diff
@@ -33,3 +39,20 @@ func init() {
 	// Fix the finality depth for testing.
 	channel.TxFinalityDepth = 3
 }
+
+func makeRoleSetups(s *ethctest.Setup, names [2]string) (setup [2]clienttest.RoleSetup) {
```

Comment:
I think `init*.go` files should only contain initialization code.  
This function would better fit into a new package `[backend/eth/client/test] setup.go` since it is generic test code for `[backend/eth/client]`.